### PR TITLE
Split ArduPilot tuning in ArduSub and ArduCopter

### PR DESCRIPTION
**Pull request by Williangalvani**
This adds ArduCopter and ArduSub options to current ArduPilot tuning page, and moves the information currently there to the ArduCopter page.

Preview [here](https://github.com/Williangalvani/qgc-user-guide/blob/splitTuning/en/SetupView/Tuning.md).

### CODE REVIEW

File: en/SetupView/tuning_ardusub.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# ArduSub Tuning
+
+![ArduSub Tuning Page](../../assets/setup/tuning/ardusub.jpg)
+
+## Basic Tuning
+
+This page allows changing the PID controller gains to better suit your vehicle and application. Changing these may help you get snappier movements for more precise movements, or a smoother response for recording cinematic footages. Adjust a parameter by moving the desired slider, or by clicking the increase/decrease buttons. There are three controllers that can be adjusted here:
```
Comment (hamishwillee) [inline en/SetupView/tuning_ardusub.md]: more precise movements > more precise control?

**Comment by hamishwillee** [comment]
Makes sense to me - modulo the comment. Is this going into the next release or is it present now?

File: en/SetupView/tuning_ardusub.md (hunk 1 above)
Comment (Williangalvani) [reply en/SetupView/tuning_ardusub.md]: I went with "Changing these may help you get a snappier response for more precise movements, or a smoother response for recording cinematic footages"

**Comment by Williangalvani** [comment]
> Makes sense to me - modulo the comment. Is this going into the next release or is it present now?

this is for next release too

**Comment by hamishwillee** [comment]
OK, so I've noticed that @DonLakeFlyer is now for some docs (essentially changes that aren't mixed with other docs) merging the content but using headings and sidebar markup to indicate that it is a daily build (still having info in the daily build record of course).

If you wish to do this and merge now that would be OK with me BUT you would need to remember to remove the daily build warnings as soon as a release happens. Given the release imminence, you might want to just hold off as we have been.

**Comment by Williangalvani** [comment]
Ok, but 3.5.0 has been released now, right?

**Comment by DonLakeFlyer** [comment]
> Ok, but 3.5.0 has been released now, right?

Yes it has.

> OK, so I've noticed that @DonLakeFlyer is now for some docs (essentially changes that aren't mixed with other docs) merging the content but using headings and sidebar markup to indicate that it is a daily build (still having info in the daily build record of course).

Not on purpose. Did I screw something up?

**Comment by hamishwillee** [comment]
@DonLakeFlyer Perhaps I missinterpret - isn't Structure Survey 2 a daily build item, but you've put it in the docs anyway - just marked up clearly as such?
I don't see this as a screwup but as an improvement. It's just not the normal model we have used in the past - new docs tending to be in the daily build page only. 

So William, is 3.5 the version that this is present in?

**Comment by Williangalvani** [comment]
> So William, is 3.5 the version that this is present in?
Yes!

**Comment by hamishwillee** [comment]
Merged then! Thanks

**Pull request merged**